iocore/net/quic/QUICLossDetector.h

```diff
@@ -36,6 +36,7 @@
 #include "QUICFrame.h"
 #include "QUICFrameHandler.h"
 #include "QUICConnection.h"
+#include "QUICPinger.h"
```

Comment:
I think we don't need to include this here. `class QUICPinger;` should be enough.

Reply:
done